addon/components/power-select/options.js

```diff
@@ -57,8 +58,8 @@ export default Component.extend({
   },
 
   // CPs
-  'aria-controls': computed('select.uniqueId', function() {
-    return `ember-power-select-trigger-${this.get('select.uniqueId')}`;
+  'aria-multiple': computed(function() {
```

Comment:
Very very minor but I wonder if this could be `computed.bool('${isArray(this.get('select.selected'))}')`

Reply:
That doesn't work.